web/src/main/java/com/bazaarvoice/emodb/web/EmoModule.java

```diff
@@ -274,8 +275,25 @@ protected void configure() {
     }
 
     private class DataStoreSetup extends AbstractModule  {
+
+        private DataStoreSetup() {
+            checkValidSystemTablePlacement();
+        }
+
+        private void checkValidSystemTablePlacement() {
+            String systemTablePlacement = _configuration.getSystemTablePlacement();
+            String systemKeyspace = systemTablePlacement.substring(0, systemTablePlacement.indexOf(':'));
+            for (CassandraConfiguration cassandraConfig : _configuration.getDataStoreConfiguration().getCassandraClusters().values()) {
+                if (cassandraConfig.getKeyspaces().containsKey(systemKeyspace)) {
+                    return;
+                }
+            }
+            throw new ProvisionException("System Table Placement references a keyspace not defined in 'cassandraKeyspaces");
+        }
+
         @Override
         protected void configure() {
+            bind(String.class).annotatedWith(SystemTablePlacement.class).toInstance(_configuration.getSystemTablePlacement());
```

Comment:
@billkalter  The reason i bound these separately was was so that the modules that were not dependent on this binding did not have access to it. But I have no issue with moving it.

Reply:
It's already bound and installed in the Common module, which means that all other installed modules already have access to it.  I checked this out and ran it with all of the bindings except the one in Common removed and all of the bindings were still satisifed.

Reply:
Whoops I didn't notice it was already globally bound.